cookiecutter.json

```diff
@@ -6,6 +6,7 @@
     "package_dir_name": "{{ cookiecutter.project_name|replace(' ', '_')|replace('-', '_')|lower }}",
     "repo_name": "{{ cookiecutter.project_name|replace(' ', '-')|lower }}",
     "project_short_description": "Python package for doing science.",
+    "maximum_supported_python_version": ["3.10", "3.11", "3.12"],
```

Comment:
change this to 3.12, 3.13, 3.14 to future proof it?  Or better, change the default to 3.12 so we can accept with just a CR.